exporter/SynthesisFusionAddin/src/APS/APS.py

```diff
@@ -65,20 +65,23 @@ def getAuth() -> APSAuth | None:
     global APS_AUTH
     if APS_AUTH is not None:
         return APS_AUTH
-    try:
-        curr_time = time.time()
+
+    currTime = time.time()
+    if os.path.exists(auth_path):
```

Comment:
Should it display a messagebox or smth?

Reply:
I'm thinking not. Just means that the user is not currently logged in / has never logged in before. But this could be up for debate.

Reply:
omg I'm dumb, I was thinking of the download path oml, totally out of the scope of this PR + smth that won't ever be undefined. 🤦‍♀️